zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/usertask/UserTaskProcessor.java

```diff
@@ -97,6 +104,20 @@ private void processCompleteTaskListener(final TypedRecord<UserTaskRecord> comma
             () -> commandProcessor.onFinalizeCommand(command, persistedRecord));
   }
 
+  private void processRejectTaskListener(final TypedRecord<UserTaskRecord> command) {
```

Comment:
```suggestion
  private void processDenyTaskListener(final TypedRecord<UserTaskRecord> command) {
```

🔧 As the `REJECT_TASK_LISTENER` intent name was updated to `DENY_TASK_LISTENER`